Sources/MapboxMaps/Foundation/Camera/CameraManager.swift

```diff
@@ -394,12 +394,21 @@ public class CameraManager {
      - Parameter animated: Indicates  whether the camera changes should be animated.
      - Parameter pitchedDrift: This hack indicates that the calling function wants to simulate drift. Therefore we need to do some additional calculations
      */
-    public func moveCamera(by offset: CGPoint? = nil, rotation: CGFloat? = nil, pitch: CGFloat? = nil, zoom: CGFloat? = nil, animated: Bool = false, pitchedDrift: Bool = false) {
+    public func moveCamera(by offset: CGPoint? = nil, rotation: CGFloat? = nil, pitch: CGFloat? = nil, zoom: CGFloat? = nil, animated: Bool = false) {
         guard let mapView = mapView else {
             assertionFailure("MapView is nil.")
             return
         }
 
+        var newPitch: CGFloat = 0
+        var pitchedDrift = false
+        if let pitchAngle = pitch {
```

Comment:
what happens if the `mapView.camera.pitch != 0` and the parameter `pitch` is `nil`? Then pitchedDrift would still be false